plugins/carla/cmake/lilv.cmake

```diff
@@ -0,0 +1,91 @@
+# base config
+if(MSVC)
+  set(carla_lilv_compile_options /wd4005 /wd4090 /wd4133 /wd4244 /wd4267 /wd4273)
```

Comment:
Too many disabled warnings, please fix warnings in code first.

Reply:
It is not my own code though, but a vendored library.
It is an older version which I can update, but I am not sure how much upstream tests for those flags in particular.

Is there a policy regarding vendored libs?

Reply:
As tytan explained above we prefer to have such libraries pre-built via `obs-deps` or available as Debian packages, because their compilation requirements don't "leak" into `obs-studio` and they stay contained within their own project. That's part of why we want to move away from vendors libraries and in-tree dependencies.

Reply:
no longer applies, since we will do carla build separately so I will keep its custom lilv as-is for now.
marking as resolved